City-of-Helsinki/hel-fi-drupal-grants
# fix: AU-1489: Application search translation fix

# [AU-1489](https://helsinkisolutionoffice.atlassian.net/browse/AU-1489)

## What was done
This pull request implements a fix for the `Applicant` / `Sökande` / `Avustuksen hakija` dropdown in the `Application search view`. Previously translations did not work properly in the dropdown, and sometimes English values would show up when viewing the application search in either Finnish or Swedish. 

## How to install
Make sure your instance is up and running on the correct branch.
  * `git checkout fix/AU-1489-application-search-translation-fix`
  * `make fresh`
  * `make drush-cr`

## How to test
- Import the translations with `drush locale:check; drush locale:update; drush cr`.
- Go to `/en/application-search` and check that the `Applicant` dropdown contains English translations.
- Go to `/sv/sok-understod` and check that the `Sökande` dropdown contains Swedish translations.
- Go to `/fi/etsi-avustusta` and check that the `Avustuksen hakija` dropdown contains Finnish translations.
- Review the code.


[AU-1489]: https://helsinkisolutionoffice.atlassian.net/browse/AU-1489?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
